src/test/java/io/pravega/connectors/flink/FlinkPravegaReaderITCase.java

```diff
@@ -49,8 +49,6 @@
 
     @BeforeClass
     public static void setupPravega() throws Exception {
-        SETUP_UTILS.setEnableAuth(true);
```

Comment:
Are we removing these lines because these are now enabled by default?

Reply:
Yes, I have moved the configuration to `SetupUtils` class so that we can control (toggle) it from one place.